<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bookmarkButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Bookmark</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="hideButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="booklist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>List</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchName">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="namelabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="namesearch">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Searching</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>edit</string>
    </property>
   </widget>
   <zorder>hideButton</zorder>
   <zorder>tableWidget</zorder>
   <zorder>addButton</zorder>
   <zorder>searchButton</zorder>
   <zorder>bookmarkButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>removeButton</zorder>
   <zorder>booklist</zorder>
   <zorder>searchName</zorder>
   <zorder>namelabel</zorder>
   <zorder>namesearch</zorder>
   <zorder>editButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPhonebook">
    <property name="title">
     <string>Phonebook</string>
    </property>
   </widget>
   <addaction name="menuPhonebook"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
